schemas/flags.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     type: 'string',
   },
   'dev-ware': {
-    desc: 'Set options for webpack-dev-middleware. e.g. --dev-ware.publicPath /',
+    desc:
+      'Set options for webpack-dev-middleware. e.g. --dev-ware.publicPath /',
```

Comment:
please keep an eye on your editor making unnecessary style changes

Reply:
I'll fix it but note that `lint-staged` is doing this.